dwave/cloud/solver.py

```diff
@@ -127,22 +144,66 @@ def sample_ising(self, linear, quadratic, **params):
 
         Returns:
             :obj:`Future`
+
+        Examples:
+            This example creates a client using the local system's default D-Wave Cloud Client
+            configuration file, which is configured to access a D-Wave 2000Q QPU, submits a
+            simple :term:`Ising` problem (opposite linear biases on two coupled qubits), and samples
+            5 times.
+
+            >>> from dwave.cloud import Client
+            >>> with Client.from_config() as client:
+            ...     solver = client.get_solver()
+            ...     u, v = next(iter(solver.edges))
+            ...     computation = solver.sample_ising({u: -1, v: 1},{}, num_reads=5)   # doctest: +SKIP
+            ...
+            >>> for i in range(5):    # doctest: +SKIP
```

Comment:
It's important to do all communication with SAPI under a live `Client` context.
On context closing (exit), workers responsible for checking problem status on SAPI and downloading of results are terminated -- and pending futures are *never* resolved.

The `for` loop in this example must be under `with` block. It wouldn't have to be only if future was resolved within the context, with e.g. `samples = computation.samples`, or `result = computation.result()`.